src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -27,6 +28,13 @@ def _environment_variables_type(value):
     return {'name': env_name, 'value': env_value}
 
 
+secrets_type = CLIArgumentType(
+    validator=validate_secrets,
+    help="space-separated secrets in 'key=value' format.",
+    nargs='*'
```

Comment:
nargs='*' means it takes 0 or more arguments. Is this what you intended?  1 or more is nargs='+'

Reply:
Good catch, I think that `+` makes more sense. I was 'inspired' by the `--tags` pattern, but that has a valid use case for 'empty' tags array.
Pending...